### PR TITLE
:children_crossing: UPDATE: sveltekit demo

### PR DESCRIPTION
**Pull request by jycouet**
- Manage the unsubscribe
- improve demo to show SSR & Options

### CODE REVIEW

File: examples/sveltekit-todo/src/routes/+page.svelte [hunk 1]
```diff
@@ -46,7 +45,20 @@
     }
   }
   async function setAllCompleted(completed: boolean) {
-    await TasksController.setAllCompleted(completed)
+    await TasksController.setAllCompleted(
```
[the remark this answers is not in the document]
Comment (jycouet) [reply examples/sveltekit-todo/src/routes/+page.svelte:48]: Yeah, it was to show pagination & options. But you are right; it's drifting a bit from the tutorial & original demo.
I'll revert `setAllCompleted`.

What do you think: 
- Adding a second backend function to handle 'selected Ids' & adding a button in the UI?
- Remove this selectedIdsComplete?

File: examples/sveltekit-todo/src/shared/tasksController.ts [hunk 2]
```diff
@@ -3,9 +3,11 @@ import { Task } from "./task"
 
 export class TasksController {
   @BackendMethod({ allowed: Allow.authenticated })
-  static async setAllCompleted(completed: boolean) {
+  static async setAllCompleted(ids: number[], completed: boolean) {
     const taskRepo = remult.repo(Task)
-    for (const task of await taskRepo.find()) {
+    const orIds = ids.map(id => { return { id: id } })
+
+    for (const task of await taskRepo.find({ where: { $or: orIds } })) {
```
[the remark this answers is not in the document]
Comment (jycouet) [reply examples/sveltekit-todo/src/shared/tasksController.ts:10]: Ho, great. 
Looking at the doc, it look like `where:{ id:{ $in:Ids } }` would work 💪
I'll look at `taskRepo.find({where:{id:ids}})` as well. Thx

File: examples/sveltekit-todo/src/routes/+page.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (noam-honig) [reply examples/sveltekit-todo/src/routes/+page.svelte:48]: whatever you prefer - I'm with you :)

[the remark this answers is not in the document]
Comment (jycouet) [reply examples/sveltekit-todo/src/routes/+page.svelte]: I prefer to stay as close as possible to the tuto.  I removed stuff.
I resolve this part.

**Pull request merged**